packages/app/src/cli/services/function/binaries.ts

```diff
@@ -9,7 +9,7 @@ import fs from 'node:fs'
 import * as gzip from 'node:zlib'
 import {fileURLToPath} from 'node:url'
 
-const FUNCTION_RUNNER_VERSION = 'v6.3.0'
+const FUNCTION_RUNNER_VERSION = 'v6.4.0'
```

Comment:
I understand why we're bumping this. Will someone who is already developing functions get the updated function-runner version? Do they have to make any manual step to move to 6.4.0 from whatever version they currently have?

Reply:
I've just triggered a snapshot so that we can validate switching versions causes it to re-download based on this version number.